code/modules/projectiles/guns/projectile/pistol/avasarala.dm

```diff
@@ -14,10 +14,11 @@
 	damage_multiplier = 1
 	penetration_multiplier = 0.9
 	recoil_buildup = 8
-	one_hand_penalty = 15
+	one_hand_penalty = 12
 	unload_sound 	= 'sound/weapons/guns/interact/hpistol_magout.ogg'
 	reload_sound 	= 'sound/weapons/guns/interact/hpistol_magin.ogg'
 	cocked_sound 	= 'sound/weapons/guns/interact/hpistol_cock.ogg'
+	fire_sound = 'sound/weapons/guns/fire/fire_silver.ogg'
```

Comment:
Do not do this

Reply:
Why

Reply:
the silver-eye sound effect? eh...the Ava is intended to be a poorly performing gun that just looks cool, no?

Reply:
Gun sounds that are uniquic to a gun should only be on that gun, in cases were people hear it to have better hear witnesses

Reply:
Except for a literal single gun and the recent Vector, no other gun in the entire server has ever had unique sounds

Reply:
Fixed the format_bound0 sprite. How do I add it to this commit?

Reply:
click push on github desktop
were it has the little arrow with the 1

Reply:
Also many guns have uniquic sounds.
Silver eye will keep its own sound as its own.